tomviz/FFTExample.cxx

```diff
@@ -0,0 +1,38 @@
+#include <vtkPointData.h>
+#include <vtkDataArray.h>
+#include <vtkImageData.h>
+#include <vtkNew.h>
+#include <vtkImageFFT.h>
+
+
+// So I think this is the basic version of what you want to do.  I'm not sure though.
+// If you don't have a vtkImageData you would need to make one (see other function
+//
+// I don't know what the other options on VTK's FFT do... And you may want to set the dimensionality to 2
+// Here are the other options:
+// http://www.vtk.org/doc/nightly/html/classvtkImageFFT.html
+void fftOfImage(vtkImageData *image)
```

Comment:
Well technically it is always 3D.  But if you want a 2D image, you just set the third dimension to be one slice thick.  And you may be able to do 2D FFTs across all the slices.  I'm guessing that is what vtkImageFFT does if you call `SetDimensionality(2)`, you'll probably want to test this to confirm.  The docs on vtkImageFFT are pretty sparse.